dist/ci/travis_script.sh

```diff
@@ -33,6 +33,7 @@ if test -z "$SUBTEST"; then
       bundle exec rails test:spider
       ;;
     linter)
+      unset DO_COVERAGE
       make -C ../../ rubocop
```

Comment:
Good spot. I saw that too. But need to check first if enabling and disabling the coverage works as I expect it.
And before that I wanted make sure that need this PR at all (aka. inspect the coverage report).